### PR TITLE
Switch to SLE12SP1 for Cloud 6 and beyond

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by aplanas** [comment]
There are some references to `$want_sles12sp1` like in L553 and references to SLE_12 repos in L1163

**Comment by dirkmueller** [comment]
@aplanas removed as well

**Comment by aplanas** [comment]
LGTM: +1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -552,7 +550,7 @@ function addcloudrubygemrepo()
             zypper ar -f http://download.suse.de/ibs/Devel:/Cloud:/Shared:/Rubygem/SLE_11_SP3/Devel:Cloud:Shared:Rubygem.repo
             ;;
         develcloud6|susecloud6|M?|Beta*|RC*|GMC*|GM6|GM6+up)
-            if [ -n "$want_sles12sp1" ]; then
+            if iscloudver 6plus || [[ ! $cloudsource =~ ^M[1-4]+$ ]]; then
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: M5, not M4.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -659,7 +657,7 @@ function get_crowbar_node()
 function get_sles12plus_node()
 {
     local target="suse-12.0"
-    [ -n "$want_sles12sp1" ] && target="suse-12.1"
+    [ iscloudver 6 ] && target="suse-12.1"
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Other changes are `6plus`, maybe use the same here for consistency?

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -1031,8 +1022,7 @@ function onadmin_prepare_cloud_repos()
                 addsles12testupdates
                 ;;
             develcloud6)
-                addsles12testupdates
-                [ -n "$want_sles12sp1" ] && addsles12sp1testupdates
+                addsles12sp1testupdates
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: No sles 12 test updates for ceph? (I'm fine with that, just asking)

File: scripts/qa_crowbarsetup.sh [hunk 4]
```diff
@@ -1273,10 +1247,7 @@ EOF
 
     if iscloudver 6plus ; then
         onadmin_prepare_sles12_repos
-        if [ -n "$want_sles12sp1" ] ; then
-            onadmin_prepare_sles12sp1_repos
-        fi
-        onadmin_prepare_sles12plus_cloud_repos
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: I'm not sure that removing that last line is fully correct, see what onadmin_create_sles12plus_repos does.

**Comment by toabctl** [comment]
+1, and needed to fix C& mkcloud....

**Pull request merged**